test/vecelement.jl

```diff
@@ -90,7 +90,7 @@ const _llvmtypes = Dict{DataType, String}(
     """
     return quote
         Base.@_inline_meta
-        Base.llvmcall($exp, Vec{$N, $T}, Tuple{Vec{$N, $T}, Vec{$N, $T}}, x, y)
+        Core.getfield(Base, :llvmcall)($exp, Vec{$N, $T}, Tuple{Vec{$N, $T}, Vec{$N, $T}}, x, y)
```

Comment:
Why is this necessary

Reply:
Because `Core.llvmcall` is actually special syntax and not a valid function. It also works as just `llvmcall`, as more commonly written.